pkg/goods/support/host-support-bundle.tmpl.yaml

```diff
@@ -73,6 +73,23 @@ spec:
       collectorName: top
       command: top
       args: ['-b', '-n', '1']
+  - run:
+      collectorName: uptime
+      command: uptime
+      args: []
```

Comment:
I think this is a duplicate it's on line 93 as well.

Reply:
good catch